File: Spring-backend/Springbackend/src/main/java/com/dravicenne/backend/models/antecedent/GynecoObstetricaux.java
package com.dravicenne.backend.models.antecedent;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import java.io.Serializable;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class GynecoObstetricaux implements Serializable {
    private String nombreGrossesse;
    private String nombreEnfant;
    private String cesarienne;
    private String autres;
}

File: Angular-frontend/src/app/Models/antecedent/gyneco-obstetrical/gyneco-obstetricaux.ts
export class GynecoObstetricaux {
  private _nombreGrossesse: string;
  private _nombreEnfant: string;
  private _cesarienne: string;
  private _autres: string;
  private _none: string;

  get none(): string {
    return this._none;
  }

  set none(value: string) {
    this._none = value;
  }

  get nombreGrossesse(): string {
    return this._nombreGrossesse;
  }

  set nombreGrossesse(value: string) {
    this._nombreGrossesse = value;
  }

  get nombreEnfant(): string {
    return this._nombreEnfant;
  }

  set nombreEnfant(value: string) {
    this._nombreEnfant = value;
  }

  get cesarienne(): string {
    return this._cesarienne;
  }

  set cesarienne(value: string) {
    this._cesarienne = value;
  }

  get autres(): string {
    return this._autres;
  }

  set autres(value: string) {
    this._autres = value;
  }
}

